config/locales/en.yml

```diff
@@ -422,18 +430,35 @@ en:
       mentions_ex1: 'Did you know you can grab someone’s attention by “mentioning” them? Simply type a @ symbol followed by the name of one of your contacts. As you start typing their name (their screen name, not their user name) it will automatically be completed by diaspora*. Hit enter or click the name in the auto-completer and you’ll notice that it has changed to their full name, and the @ symbol has disappeared. Don’t worry, though; this will become an @mention once you have posted the message. The person you are mentioning will receive a notification of the mention on their notification page, and by email if they have asked to receive email notification of mentions.'
       mentions_ex2: 'Likewise other people can @mention you, and you’ll receive a notification in the header bar (and by email if you have set that option).'
       mentions_ex3: 'Note that you can only @mention people in your aspects; and only in posts, not in comments.'
+
       format: 'Text formatting'
-      format_ex: 'There’s a lot you can do to make your text more varied: bold, italics, large headings and so on. diaspora* uses a simple mark-up system called Markdown. It’s not in the scope of this introduction, however, so we explore how to use Markdown in %{format_link}.'
-      format_link_text: 'this tutorial'
+      format_ex1: 'There’s a lot you can do to make your text more varied: bold, italics, large headings and so on.'
+	  format_ex2: 'To add <em>italic</em>, <strong>bold</strong>, or code, highlight the text you want to format and click the relevant button from the row above the publisher window. The Markdown code for this formatting will be placed around this portion of text.'
+	  format_ex3: 'To create a heading, list or quotation, place your cursor on the line you want to format, and press the relevant button.'
```

Comment:
Thanks. Don't know how that one slipped through.

You can ignore this branch for now. I'll use it to commit updates to make sure they aren't lost on my machine (already happened once today), and will shout 'help' when I'm ready for review. Appreciate your attention, but don't want to waste your time until then.